src/trusted/service_runtime/nacl_copy.c

```diff
@@ -35,6 +36,9 @@ int NaClCopyInFromUserAndDropLock(struct NaClApp *nap,
                                   size_t         num_bytes) {
   uintptr_t src_sys_addr;
 
+  /* We don't need to check protections here
+   * this is used by nacl_syscall_handlers, and we can check the actual args later
+   */
```

Comment:
You either need to check here or check before, as otherwise the memcpy can fault. So either update the comment or the code.